src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -564,6 +572,8 @@ class SpfxDoctorCommand extends BaseProjectCommand {
     );
   }
 
+  private resultsObject: SpfxDoctorCheck[] = [];
```

Comment:
I think we should merge this with what's currently called the `fixes` array. It's purpose may become somewhat broader. We could add all things that need to be in the output to this array. 

We could decide at the end what we want to log based on the items in this array and the output mode... idea?